var/spack/repos/builtin/packages/kokkos/package.py

```diff
@@ -281,3 +285,47 @@ def cmake_args(self):
         options.append('-DBUILD_SHARED_LIBS=%s' % ('+shared' in self.spec))
 
         return options
+
+    test_script_relative_path = "scripts/spack_test"
```

Comment:
Can we document approximately here what order/when we expect these functions to get invoked?

Reply:
I would propose omitting documentation as people who'll be reading this probably know that. ```"@run_after('install')"``` runs after the install step where we move test source files to the .spack/test dir. During ```spack test run```, we build and run.